prefq/examples/imitation_preference_comparisons.py

```diff
@@ -42,6 +209,11 @@
     ),
     env=venv,
     n_steps=2048 // venv.num_envs,
+    seed=0,
+    batch_size=64,
+    ent_coef=0.0,
+    learning_rate=0.0003,
+    n_epochs=10,
```

Comment:
Why these changes?

Reply:
these changes have been copied from one of the examples provided by the authors of the PrefCollectGatherer: 

https://github.com/HumanCompatibleAI/imitation/blob/4dbf20c6d42286377901dd538cd6c5001225fa40/docs/tutorials/5b_train_preference_comparisons_with_synchronous_human_feedback.ipynb

Reply:
I think those changes should not be relevant for the imitation integration.